src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1383,13 +1383,21 @@ void MulticopterPositionControl::control_auto(float dt)
 	if (_pos_sp_triplet.current.valid) {
 
 		//only project setpoints if they are finite, else use current position
+		static bool first_nonfinite = true;
```

Comment:
Why is the state required here? And would it maybe be more accurate to set the position to lat/lon of home in the navigator land.cpp instead?